docs/testcases/UI/Pipe_CLI/1948_launch_run_as_different_user/1948_1_check_pipe_cli_admin_run_as_user.md

```diff
@@ -0,0 +1,24 @@
+# Check Pipe CLI Admin Run As User
+
+Test verifies that an admin can launch a run as a different user using pipe CLI.
+
+**Prerequisites**:
+- Admin user
+- Regular user
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Tools** page | |
+| 3 | Select test tool | |
+| 4 | Run tool with *Custom settings* | |
+| 5 | At the Runs page, click the just-launched run | |
+| 6 | Wait until the **SSH** hyperlink appears | |
+| 7 | Click the **SSH** hyperlink | |
+| 8 | In the opened tab, enter and perform the `pipe run -di <tool>:latest -u <REGULAR USER> -y` command, where `<tool>` is any tool name with group, `<REGULAR USER>` is the Regular user from the Prerequisites | The output contains `Pipeline run scheduled with RunId: <runID>` |
```

Comment:
> where `<tool>` is any tool name with group

Add the note like `tool shall be accessible for the <REGULAR USER> for the running`